namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-stg/resources/ecr.tf

```diff
@@ -0,0 +1,31 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "cica_ecr_credentials" {
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=3.2"
```

Comment:
Can you please upgrade to the latest version of the module : 3.4 
https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/releases

Reply:
Updated config reference to 3.4 now